templates/zerver/help/editing-past-messages.md

```diff
@@ -0,0 +1,33 @@
+### Editing past messages
+
+ * If you want to edit a past message of your own after it has been
+   posted, hover your mouse over the message. You should see a
+   little pencil ![pencil](/static/images/help/pencil.png) at the end
+   of the message and a “down-chevron”
+   ![down chevron](/static/images/help/down_chevron.png) near the
+   upper right corner of the message. Clicking either one of these
+   will give you editing options.
+   
+ * It is possible that your chat is configured to set a limit on the time you may
+   edit a message, e.g. 10 minutes. As soon as that limit has passed, the little pencil
+   ![pencil](/static/images/help/pencil.png)  changes to a file icon 
+   <img src="/static/images/help/file.png" width = "18">. Depending on the configuration, clicking on it might allow you
+   to change the topic, 
+   like shown in this message 
+   
+   <img src="/static/images/help/message_time_limit_passed.png" width="480">
+
+ * You cannot entirely delete a message. When you remove all the text of a message and save it,
+   Zulip will automatically insert a "(deleted)" into your message. 
+   The fact that there was once a message there will still be
+   evident, and the time stamp of the original posting date and time
+   will still be shown.
+
```

Comment:
Again, adding a section header may help:

## Deleting Messages
You cannot entirely delete a message...